### PR TITLE
[tools] Translate strings using DeepL free API + Google Translate

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: tools/python/translate.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    exit(1)
+  return key
+
+def google_translate(text):
+  fromTo = 'en:'
+  for lang in GOOGLE_TARGET_LANGUAGES:
+    fromTo += lang + '+'
+  # Remove last +
+  fromTo = fromTo[:-1]
+  res = subprocess.run(['trans', '-b', '-no-bidi', fromTo, text], text=True, capture_output=True)
```
Comment (rtsisyk) [inline tools/python/translate.py:76]: It would be better to call the API directly instead of relying on additional external tool.

**Review by rtsisyk — COMMENTED** [review]
I contributed a similar script before (191434038fb31bb1102c8cf49fad4e7a33308dfa), but was removed (c452ffb7857f4cfc457527bfee1eea46264755c8) without providing any working alternative.

File: tools/python/translate.py (hunk 1 above)
Comment (biodranik) [reply tools/python/translate.py:76]: Then we need to solve additional issues with API keys. You can always improve this working version later if you have enough time.

**Comment by biodranik** [comment]
> I contributed a similar script before (https://github.com/organicmaps/organicmaps/commit/191434038fb31bb1102c8cf49fad4e7a33308dfa), but was removed (https://github.com/organicmaps/organicmaps/commit/c452ffb7857f4cfc457527bfee1eea46264755c8) without providing any working alternative.

Your script wasn't similar. It used only Google Maps for translations, that are:
1) Worse than DeepL
2) Screw up Portuguese Brazil
3) Based on outdated and not actively supported external python library

**Comment by biodranik** [comment]
PTAL

**Comment by rtsisyk** [comment]
> 

OK. Understand.

File: tools/python/translate.py (hunk 1 above)
Comment (rtsisyk) [reply tools/python/translate.py:77]: https://github.com/organicmaps/organicmaps/commit/c452ffb7857f4cfc457527bfee1eea46264755c8

**Comment by biodranik** [comment]
@rtsisyk @vng PTAL

**Review by vng — APPROVED** [review]

**Pull request merged**